package.json

```diff
@@ -30,19 +30,13 @@
     "lint-fix": "eslint --fix lib/*.js examples/*.js"
   },
   "dependencies": {
-    "@iota/core": "1.0.0-beta.11",
-    "@iota/validators": "1.0.0-beta.11",
-    "@iota/bundle": "1.0.0-beta.11",
-    "@iota/bundle-validator": "1.0.0-beta.11",
-    "@iota/checksum": "1.0.0-beta.11",
-    "@iota/converter": "1.0.0-beta.11",
-    "@iota/curl": "1.0.0-beta.11",
-    "@iota/http-client": "1.0.0-beta.11",
-    "@iota/kerl": "1.0.0-beta.11",
-    "@iota/pad": "1.0.0-beta.11",
-    "@iota/signing": "1.0.0-beta.11",
-    "@iota/transaction": "1.0.0-beta.11",
-    "@iota/transaction-converter": "1.0.0-beta.11",
+    "@iota/core": "1.0.0-beta.17",
```

Comment:
Would you upgrade to `1.0.0-beta.18` or `1.0.0-beta.17`?
https://github.com/iotaledger/iota.js/releases